### ISSUE TITLE
"build" and "dev" commands crash with "yarn link" dependency

### ISSUE THREAD

**Issue by aleclarson** [reframejs/reframe#52]
Repro: https://github.com/aleclarson/repro/tree/reframe-1

When `yarn link` is used, both `reframe build` and `reframe dev` crash with the following error:
```
****************************************
**************** Error *****************
****************************************
~/reframe/node_modules/@babel/runtime/helpers/esm/extends.js:1
(function (exports, require, module, __filename, __dirname) { export default function _extends() {
                                                              ^^^^^^

SyntaxError: Unexpected token export
    at new Script (vm.js:85:7)
    at createScript (vm.js:266:10)
    at Object.runInThisContext (vm.js:314:10)
    at Module._compile (internal/modules/cjs/loader.js:698:28)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:749:10)
    at Module.load (internal/modules/cjs/loader.js:630:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:570:12)
    at Function.Module._load (internal/modules/cjs/loader.js:562:3)
    at Module.require (internal/modules/cjs/loader.js:667:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.require (~/reframe/dist/nodejs/webpack:/external "@babel/runtime/helpers/esm/extends":1:1)
    at call (~/reframe/dist/nodejs/webpack:/webpack/bootstrap:19:1)
    at Module.<anonymous> (~/reframe/dist/nodejs/webpack:/external "@babel/runtime/helpers/esm/objectWithoutPropertiesLoose":1:1)
    at call (~/reframe/dist/nodejs/webpack:/webpack/bootstrap:19:1)
    at Object.<anonymous> (~/reframe/dist/nodejs/webpack:/external "@babel/runtime/helpers/esm/objectWithoutPropertiesLoose":1:1)
    at call (~/reframe/dist/nodejs/webpack:/webpack/bootstrap:19:1)
```

More details in the repro

**Comment by brillout** [comment]
That's because somehow webpack doesn't write the bundle to the disk. Not sure why, I'll dig deeper later.

Thanks a lot. Opening issues for bugs are immensely valuable, thanks!

**Comment by brillout** [comment]
@aleclarson The problem is that react-spring's `main` field of `package.json` points to `web.cjs.js` instead of `dist/web.cjs.js`.

I improved Reframe's error reporting and I will publish a new version today/tomorrow.

**Comment by aleclarson** [comment]
Ah, that makes sense. Thanks!

**Comment by brillout** [comment]
Better error reporting published in `v0.3.2`.
